actnow/profiles/models.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from phonenumber_field.modelfields import PhoneNumberField
+
+from .validators import validate_social_media_link
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(get_user_model(), on_delete=models.CASCADE)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    bio = models.TextField(
+        verbose_name="About me", max_length=255, blank=True, null=True
+    )
+    profile_photo = models.FileField(max_length=30)
```

Comment:
```suggestion
    photo = models.FileField(max_length=30)
```
We know it's a profile.